examples/jsm/renderers/common/RenderObject.js

```diff
@@ -72,6 +73,18 @@ export default class RenderObject {
 
 	}
 
+	getAttributeHash( index ) {
```

Comment:
Maybe we should rename these methods to `*UUID`, not `*Hash`?

Reply:
Yes, I made it :) [Attributes: Rename .getHash() to .getUUID()](https://github.com/mrdoob/three.js/pull/26186/commits/809d82175eee14f287ae1de6362168b24379e1b7)